shell/platform/android/io/flutter/plugin/platform/PlatformViewsController.java

```diff
@@ -105,6 +105,9 @@
   // Tracks whether the flutterView has been converted to use a FlutterImageView.
   private boolean flutterViewConvertedToImageView = false;
 
+  // Whether the render surface should be converted when a PlatformView is added to FlutterView.
+  private boolean convertRenderSurfaceDisabled = false;
```

Comment:
Can this be better documented? Converted from what to what?

Reply:
Done. Let me know if the explanation makes sense. I wrote it with my understanding of the conversion

Reply:
I'd suggest something like: `syncronizeFlutterToAndroidViewHierarchy`.  it's probably long, but it explains what the value does.